### DISCUSSION TITLE
CUBLAS initialization failed in example 24

### DISCUSSION THREAD

**Discussion by jeff86223** [IBM/aihwkit#487] [General]
## Description

Execute Example 24 with CUBLAS initialization error.
Is there any way to solve it? 
Thanks

![2023-02-24_19-52](https://user-images.githubusercontent.com/29406633/221172819-ec20e6f8-e9a9-46cb-85b0-3e975a390c4c.png)

## Other information

* Pytorch version: v1.12.0
* OS: Ubuntu 20.04
* Python version: 3.8.10
* CUDA: 11.3
* CUDNN v8

**Comment by maljoras** [comment]
Hi @jeff86223,
I know this is a fairly uninformative error message but it is typical a memory issue.  Your GPU card might not support enough memory. One way to reduce the memory usage is to use lower batch size.  Unfortunately, memory gets filled up quite quickly for larger DNNs.

**Comment by jeff86223** [comment]
Hi @maljoras ,

I will try to run it on a more advanced GPU, thanks!!
